app/controllers/products_controller.rb

```diff
@@ -0,0 +1,14 @@
+class ProductsController < ApplicationController
+   skip_before_action :authenticate_user!, only: [ :index ]
+
+  def index
+    @products = Product.all
+    @drones = Product.where(productable_type: 'Drone')
+    @audios = Product.where(productable_type: 'Audio')
+    @lightings = Product.where(productable_type: 'Lighting')
+    @stabilizers = Product.where(productable_type: 'Stabilizer')
+    @tripods = Product.where(productable_type: 'Tripod')
```

Comment:
Nice Kat, for this I thought something less hardcoded using params or we'll see. But is a good solutionon to start 💪